src/index.js

```diff
@@ -0,0 +1,39 @@
+const Koa = require('koa')
+const koaStatic = require('koa-static')
+const IlpNode = require('ilp-node')
+
+if (process.argv.length < 6) {
+  throw new Error('Usage: node src/index.js data/stats.json data/creds.json public/ 8600')
+}
+
+const statsFile = process.argv[2]
```

Comment:
I think these should be environment variables instead of command line arguments, so it's harder to mix them up